### ISSUE TITLE
As a SM, I need to be able to leave notes about accessibility for SO/CE

### ISSUE THREAD

**Issue by kreynen** [CuBoulder/express#987]
## Context

We've discussed this functionality in FIT-461 for notes about content, but I'd like to extend the work that has already been done for that feature to include notes left by admin level users about accessibility issues.  We need to talk to Kosta about using this feature.  Ideally the SO/CE would have the option of reporting back that they believe the issue has been fixed.  Those "needs review" requests could feed into Atlas and generate a report there.  We still need to figure out how we would deal with the staffing needs for those reviews.  We also need to know if the site has a pending launch.

## Process
1. Login to site as a ADMIN, DEV, and possible OIT Accessibility Staff
2. Navigate to the path with the issue
3. Click Add Note (not sure what the UI looks like)
4. Enter "Kevin, don't do that" in the body field
5. Click save

## Expected result

Not appears for authenticated SO/CE when viewing the page.

## Current result

NA

## Possible Fix

https://www.drupal.org/project/admin_notes/ has a new maintainer and D8 branch so we should evaluate that again.

**Comment by kreynen** [comment]
The type of notes @ekellagher would be leaving https://github.com/CuBoulder/express/issues/868#issuecomment-285742259

**Comment by kreynen** [comment]
Build on https://github.com/CuBoulder/cu_editor_note_bundle

**Comment by cathysnider** [comment]
https://www-dev.colorado.edu/p1f8e5fe52e8/

Notes after reviewing the bundle:

In Disabling Editor Notes, I am skeptical that the value added in having notes turned off, is worth the problems that will come from not having the notes there. One Site Owner can disable the notes, and the rest of the editors will have no idea that there is important information that needs to be read (unless they are cognizant that they should check the Content area for the Editor Notes tab, scroll through all the notes to see if the page they are concerned with has one). 

There should be a way to sort notes by type, since we are collecting that information. A site editor will want to review all 'Accessibility' notes, for example, to make sure the issues have been taken care of.

**Comment by kevincrafts** [comment]
There are are many types of notes that fall into two classes: regular notes and admin notes.

Regular notes can be added, edited, and deleted by editors. Administrator notes, which would be used on our team to leave critical information about accessibility, etc, cannot be deleted by site owners or content editors.

I've added a type filter to the view.

**Comment by cathysnider** [comment]
Acceptance Criteria (this list is a work in progress )
- [ ] Site Owners can NOT add or delete an Admin note
- [ ] Site Owners CAN view an Admin note.
- [ ] users with Admin level permissions CAN add, delete and view Admin notes.
